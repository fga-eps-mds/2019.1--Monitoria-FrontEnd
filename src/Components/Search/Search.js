import React, { Component } from 'react';
import { Grid} from '@material-ui/core' ;
import AppBar from './AppBarSearch';
import Paper from './PaperSearch';


class Search extends Component {
    
  render() {
      
    return (
        <div >
            <Grid container  justify="center" alignItems="stretch">
                <AppBar/>
            </Grid>    
            <Grid container  justify="center" direction="column" alignItems="center" spacing="16" style={{ padding: 80 }}>
                <Grid item >
                    <Paper/>
                </Grid>
            </Grid>
        </div>
    );   
  }
}

export default Search;