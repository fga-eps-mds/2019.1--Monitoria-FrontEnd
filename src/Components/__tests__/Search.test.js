import React from 'react';
import Search from '../Search/Search';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';
import firebase from 'firebase';

firebase.initializeApp({
  apiKey: '...',
  authDomain: '...',
  databaseURL: '...',
  projectId: '...',
  storageBucket: '...',
  messagingSenderId: '...',
});

Enzyme.configure({adapter: new Adapter()});

const props = {
    state: {
        
    }
  
};
test('renders correctly' , () => {
  const tree = renderer
  .create(<Search {...props} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});

