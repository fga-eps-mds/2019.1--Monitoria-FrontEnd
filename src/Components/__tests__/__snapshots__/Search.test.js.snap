// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Search component Test if Search renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Search />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <WithStyles(Grid)
          alignItems="stretch"
          container={true}
          justify="center"
        >
          <WithStyles(SearchAppBar) />
        </WithStyles(Grid)>,
        <WithStyles(Grid)
          alignItems="center"
          container={true}
          direction="column"
          justify="center"
          spacing="16"
          style={
            Object {
              "padding": 80,
            }
          }
        >
          <WithStyles(Grid)
            item={true}
          >
            <WithStyles(PaperSheet) />
          </WithStyles(Grid)>
        </WithStyles(Grid)>,
        <WithStyles(Grid)
          container={true}
        >
          <WithStyles(Grid)>
            <WithStyles(IconLabelTabs) />
          </WithStyles(Grid)>
        </WithStyles(Grid)>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "alignItems": "stretch",
          "children": <WithStyles(SearchAppBar) />,
          "container": true,
          "justify": "center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "alignItems": "center",
          "children": <WithStyles(Grid)
            item={true}
          >
            <WithStyles(PaperSheet) />
          </WithStyles(Grid)>,
          "container": true,
          "direction": "column",
          "justify": "center",
          "spacing": "16",
          "style": Object {
            "padding": 80,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(PaperSheet) />,
            "item": true,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Grid)>
            <WithStyles(IconLabelTabs) />
          </WithStyles(Grid)>,
          "container": true,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(IconLabelTabs) />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <WithStyles(Grid)
            alignItems="stretch"
            container={true}
            justify="center"
          >
            <WithStyles(SearchAppBar) />
          </WithStyles(Grid)>,
          <WithStyles(Grid)
            alignItems="center"
            container={true}
            direction="column"
            justify="center"
            spacing="16"
            style={
              Object {
                "padding": 80,
              }
            }
          >
            <WithStyles(Grid)
              item={true}
            >
              <WithStyles(PaperSheet) />
            </WithStyles(Grid)>
          </WithStyles(Grid)>,
          <WithStyles(Grid)
            container={true}
          >
            <WithStyles(Grid)>
              <WithStyles(IconLabelTabs) />
            </WithStyles(Grid)>
          </WithStyles(Grid)>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "alignItems": "stretch",
            "children": <WithStyles(SearchAppBar) />,
            "container": true,
            "justify": "center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "alignItems": "center",
            "children": <WithStyles(Grid)
              item={true}
            >
              <WithStyles(PaperSheet) />
            </WithStyles(Grid)>,
            "container": true,
            "direction": "column",
            "justify": "center",
            "spacing": "16",
            "style": Object {
              "padding": 80,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <WithStyles(PaperSheet) />,
              "item": true,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <WithStyles(Grid)>
              <WithStyles(IconLabelTabs) />
            </WithStyles(Grid)>,
            "container": true,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <WithStyles(IconLabelTabs) />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
